layouts/partials/components/carousel.html

```diff
@@ -0,0 +1,54 @@
+<section>
+	<div id="tinyslider-carousel">
+		{{ range $.Params.carousel }}
+		<div class="item">
+			<div>
+				{{ if reflect.IsMap . }}
+				{{ $image := partial "utils/get-image" (dict "context" $ "url" .image "keyword" .image) }}
```

Comment:
I have added this get-image function calls (3a07d8f), yet the URL it produces is wrong as images are in public/posts yet this function resolves it to root .

Reply:
As we discussed before, there is a mistake in organizing images in content folder. You can create a folder named `carousel` in this case and rename `carousel.md` to `index.md` and move it to carousel folder along with those images. And images will be in `localhost:1313/posts/carousel/carousel1.jpg` for example. The front matter should be `carousel1.jpg` for example. Please note that the images you uploaded is `jpg`, but the filename you entered is `png`.

This is called [Leaf Bundles](https://gohugo.io/content-management/page-bundles/#leaf-bundles) in Hugo.

And if you have done that, `get-image` should give you a correct result as I tested in my machine.

Reply:
Done this in 39b7b9c